test/benchmarks/material/checkbox/BUILD.bazel

```diff
@@ -0,0 +1,28 @@
+load("@npm_angular_dev_infra_private//benchmark/component_benchmark:component_benchmark.bzl", "component_benchmark")
+
+genrule(
+    name = "indigo-pink",
+    srcs = ["//src/material/core:theming/prebuilt/indigo-pink.css"],
+    outs = ["styles.css"],
+    cmd = "cp $< $@",
+)
```

Comment:
We should be able to eliminate this genrule if we serve the asset from the right path.

Reply:
Yes but then we'd need to write our own `index.html` to point to it. The default one just looks for a `styles.css`

Reply:
`ts_devserver` automatically injected assets properly in the past. Is that still the case? 

Ideally, the benchmark rule would not transitively rely on `styles.css` to be present and assets would be injected automatically. That is related to your TODO below.

Reply:
If the default `index.html` is being used and `styles.css` is not provided, a default blank one is used so that no errors are thrown.

The solution we have decided to go is to just set the view encapsulation of `CheckboxBenchmarkApp` to none so that its styles are applied to the whole page, and then make provide `indigo-pink.scss` via `styleUrls`.

This way we don't need to provide an `index.html` nor `styles.css` and can just use the default ones.

Reply:
Yeah, that solution for now makes sense, but I think long-term the component benchmark rule should simply rely on automatic asset injection. That will simplify things a lot. The default `index.html` could still be used, but one would be able to add assets without having to provide a custom index file at all, or by not having to disable view encapsulation.

We can chat about this in the future. Doesn't sound like a big deal for now. Just an idea 😄